### ISSUE TITLE
Synchronized prevents ability to do A/V syncmanually

### ISSUE THREAD

**Issue by TheSashmo** [CESNET/UltraGrid#420]
It seems that the synchronized option blocks doing manual adjustments for lip sync in both console mode and control port mode.

Measuring a local machine encode decode test, and when using drift_fix I can adjust the accuracy and get it within one field of accuracy of +/- 16ms which is spot on for my need.

But when using synchronized its stuck at +74ms not matter if I put into the command line for --audio-delay=-70, or <> to adjust in console or control port av-delay -70, nothing adjusts it.

If I change the default option of synchronized to synchronized=8,10 or any other value its still stuck at the same value as well and I can not make any adjustments in the accuracy.

When I use drift_fix I am able to dial it in to an acceptable value, and it sits there, but I get occasional buffer overflow or underflows which causes problem in the decode output.  The moment I apply the synchronized it seems to stabilize the overflow underflow errors, but looses accuracy.

Suggestions?

**Comment by alatteri** [comment]
maybe it is out of sync at the encode step, and the stream on the receiver is actually playing back synchronized, albeit with an inherent slip baked into the stream.

For instance, if in your compression parameters, you used `:threads=3`, this would equate to the video being 3 frames behind the audio, baked into the outgoing stream.

Inaddition.  I don't have time to confirm this right now, but in the configuration of our Decklink based receiver boxes, I set `--audio-delay -20`, so at least at some point in the past, this was working.

**Comment by TheSashmo** [comment]
Yeah if you are using synchronized now, the audio delay dose not work at all.  I tested on continuous build from today.  No matter what you set it (for my setup) always 74ms ahead.  The audio delay at the encoder never worked, I was told a way back from someone in the tickets that it never worked and only works on the decoder side.

**Comment by alatteri** [comment]
`--audio-filter delay:N:frames`

**Comment by TheSashmo** [comment]
Yeah this is a alternate way to solve it. Thanks @alatteri
